miso-web/src/main/webapp/scripts/bulk_sample.js

```diff
@@ -1095,6 +1095,27 @@ BulkTarget.sample = (function ($) {
           includeSaved: targetCategory === "Tissue Processing",
           precision: 14,
           scale: 10,
+          description: "Initial concentration of cells in the sample at the time of receipt",
+        },
+        {
+          title: "Target Cell Recovery",
+          type: "int",
+          data: "targetCellRecovery",
+          min: 0,
+          include: show["Tissue Processing"] && !show["Stock"],
```

Comment:
This will have the following effects:

- When creating tissue processing, it will show for the tissue processing level
- When creating stock, it will show for the stock level
- When creating aliquots, it will show for both the tissue processing and stock levels

We don't want it showing up twice, and if we're creating aliquots, it should probably be the lower-level stock field that we show. Can be fixed with:

```
include: targetCategory === "Tissue Processing"
```